internal/build/job/manager.go

```diff
@@ -101,3 +128,13 @@ func (jbm *jobManager) Sync(ctx context.Context, mod kmmv1beta1.Module, m kmmv1b
 		return build.Result{}, fmt.Errorf("unknown status: %v", job.Status)
 	}
 }
+
+func (jbm *jobManager) checkIfImageExists(ctx context.Context, mod *kmmv1beta1.Module, km *kmmv1beta1.KernelMapping, imageName string) (bool, error) {
```

Comment:
This is a duplicate of the method of the same name in `controllers/module_reconciler.go`. I don't know where would be a good location, though.

Reply:
Yes, valid point. I'm currently thinking about where this could be hosted, that's why it's still in both places. I'll come back to this one.

Reply:
I moved this method under the `module` package, [here](https://github.com/kubernetes-sigs/kernel-module-management/pull/156/files#diff-4c273cc04ac4611a608fe8e4037b9834c162daba892070611f56ffc83e47fc4dR49). What do you think?